src/formattedcode/writers.py

```diff
@@ -31,16 +31,20 @@
 import os
 from os.path import abspath
 
+import pluggy
+
 from formattedcode.format import as_template
 from formattedcode.format import as_html_app
 from formattedcode.format import create_html_app_assets
 from formattedcode.format import HtmlAppAssetCopyWarning
 from formattedcode.format import HtmlAppAssetCopyError
 
+from plugincode import hookspec
+
 
 def write_formatted_output(
         scanners, files_count, version, notice, scanned_files,
-        format, options, input, output_file, _echo):
+        format, options, input, output_file, _echo, plugin_formats, pm):
```

Comment:
Do you mean adding a new marker for `format_plugins`? Currently the formatting plugins are handled by the `post_scan` marker, according to that `post_scan_plugins` make sense IMO.